RAKE/RAKE.py

```diff
@@ -4,8 +4,11 @@
 # Automatic keyword extraction from indi-vidual documents.
 # In M. W. Berry and J. Kogan (Eds.), Text Mining: Applications and Theory.unknown: John Wiley and Sons, Ltd.
 
+from __future__ import absolute_import
 import re
 import operator
+import six
+from six.moves import range
```

Comment:
this will have to be added to setup.py or however to make pip auto install it as a dependency